kubernetes/samples/scripts/create-weblogic-domain/domain-home-on-pv/wlst/istio-fix-domain.py

```diff
@@ -0,0 +1,131 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+def getEnvVar(var):
+  val=os.environ.get(var)
+  if val==None:
+    print "ERROR: Env var ",var, " not set."
+    sys.exit(1)
+  return val
+
+domain_uid                   = getEnvVar("DOMAIN_UID")
+server_port                  = int(getEnvVar("MANAGED_SERVER_PORT"))
+domain_path                  = getEnvVar("DOMAIN_HOME")
+cluster_name                 = getEnvVar("CLUSTER_NAME")
+admin_server_name            = getEnvVar("ADMIN_SERVER_NAME")
+admin_server_name_svc        = getEnvVar("ADMIN_SERVER_NAME_SVC")
+admin_port                   = int(getEnvVar("ADMIN_PORT"))
+domain_name                  = getEnvVar("DOMAIN_NAME")
+t3_channel_port              = int(getEnvVar("T3_CHANNEL_PORT"))
+t3_public_address            = getEnvVar("T3_PUBLIC_ADDRESS")
+number_of_ms                 = int(getEnvVar("CONFIGURED_MANAGED_SERVER_COUNT"))
+cluster_type                 = getEnvVar("CLUSTER_TYPE")
+managed_server_name_base     = getEnvVar("MANAGED_SERVER_NAME_BASE")
+managed_server_name_base_svc = getEnvVar("MANAGED_SERVER_NAME_BASE_SVC")
+domain_logs                  = getEnvVar("DOMAIN_LOGS_DIR")
+script_dir                   = getEnvVar("CREATE_DOMAIN_SCRIPT_DIR")
+
+# Update Domain for Istio
+readDomain(domain_path)
+
+cd('/Servers/' + admin_server_name)
+create('istio-probe', 'NetworkAccessPoint')
```

Comment:
Should this be http-probe.  The istio convention is to use a prefix of http- when using the http protocol.

Reply:
The "istio-" prefix is used as condition in introspectDomain.py to skip overriding listen address in sitconfig. I had ever tried istio-http-probe, but it doesn't work as a wls channel name. Odd!